go.mod1

```diff
@@ -4,39 +4,56 @@ go 1.13
 
 require (
 	cloud.google.com/go/storage v1.6.0
+	github.com/BurntSushi/toml v0.3.1
+	github.com/DATA-DOG/go-sqlmock v1.5.0
 	github.com/aws/aws-sdk-go v1.35.3
-	github.com/cheggaaa/pb/v3 v3.0.4
+	github.com/carlmjohnson/flagext v0.20.2
+	github.com/cheggaaa/pb/v3 v3.0.5
 	github.com/cheynewallace/tabby v1.1.0
+	github.com/cockroachdb/pebble v0.0.0-20201023120638-f1224da22976
 	github.com/coreos/go-semver v0.3.0
+	github.com/docker/go-units v0.4.0
 	github.com/fsouza/fake-gcs-server v1.19.0
 	github.com/go-sql-driver/mysql v1.5.0
 	github.com/gogo/protobuf v1.3.1
 	github.com/golang/mock v1.4.4
 	github.com/google/btree v1.0.0
 	github.com/google/uuid v1.1.1
+	github.com/joho/sqltocsv v0.0.0-20190824231449-5650f27fd5b6
 	github.com/opentracing/opentracing-go v1.1.0
 	github.com/pingcap/check v0.0.0-20200212061837-5e12011dc712
 	github.com/pingcap/errors v0.11.5-0.20201126102027-b0a155152ca3
 	github.com/pingcap/failpoint v0.0.0-20200702092429-9f69995143ce
 	github.com/pingcap/kvproto v0.0.0-20201215060142-f3dafca4c7fd
 	github.com/pingcap/log v0.0.0-20201112100606-8f1e84a3abc8
 	github.com/pingcap/parser v0.0.0-20201222091346-02c8ff27d0bc
-	github.com/pingcap/tidb v1.1.0-beta.0.20201229090316-0217a00191eb
+	github.com/pingcap/tidb v1.1.0-beta.0.20210105101819-f55e8f2bf835
 	github.com/pingcap/tidb-tools v4.0.9-0.20201127090955-2707c97b3853+incompatible
 	github.com/pingcap/tipb v0.0.0-20201209065231-aa39b1b86217
 	github.com/prometheus/client_golang v1.5.1
+	github.com/prometheus/client_model v0.2.0
+	github.com/shurcooL/httpgzip v0.0.0-20190720172056-320755c1c1b0
 	github.com/sirupsen/logrus v1.6.0
 	github.com/spf13/cobra v1.0.0
 	github.com/spf13/pflag v1.0.5
 	github.com/tikv/pd v1.1.0-beta.0.20201125070607-d4b90eee0c70
+	github.com/xitongsys/parquet-go v1.5.5-0.20201110004701-b09c49d6d457
+	github.com/xitongsys/parquet-go-source v0.0.0-20200817004010-026bad9b25d0
 	go.etcd.io/etcd v0.5.0-alpha.5.0.20200824191128-ae9734ed278b
+	go.uber.org/atomic v1.7.0
 	go.uber.org/multierr v1.6.0
 	go.uber.org/zap v1.16.0
+	golang.org/x/net v0.0.0-20200904194848-62affa334b73
 	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	golang.org/x/sync v0.0.0-20201020160332-67f06af15bc9
+	golang.org/x/sys v0.0.0-20200905004654-be1d3432aa8f
+	golang.org/x/text v0.3.4
 	google.golang.org/api v0.22.0
 	google.golang.org/grpc v1.27.1
+	modernc.org/mathutil v1.1.1
 	sourcegraph.com/sourcegraph/appdash v0.0.0-20190731080439-ebfcffb1b5c0
 )
 
+replace github.com/pingcap/tidb-lightning => github.com/overvenus/tidb-lightning v5.0.0-rc.0.20210129142610-fe7a7a431b0c+incompatible
```

Comment:
?

Reply:
The fork updates BR in order to fix `make build`.

https://github.com/overvenus/tidb-lightning/commit/fe7a7a431b0cc0cd90d5be86b00272319383f458